### PR TITLE
Adding Pause and Resume Auto Code Scans Buttons to status bar menu

### PR DESCRIPTION
**Pull request by ashishrp-aws**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Description

Added Pause/Resume Button for Auto Code Scans. No tests were added as there were no prior tests for Pause/Resume Auto Code Recommendations.

## Checklist
- [x] My code follows the code style of this project
- [x] I have added tests to cover my changes
- [x] A short description of the change has been added to the **[CHANGELOG](https://github.com/aws/aws-toolkit-jetbrains/blob/master/CONTRIBUTING.md#contributing-via-pull-requests)** if the change is customer-facing in the IDE.
- [x] I have added metrics for my changes (if required)
 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .changes/next-release/feature-2152796a-9dd7-4889-9d00-3f56da0b2002.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "feature",
+  "description" : "Adding Pause and Resume Auto Code Scans Buttons to status bar menu"
+}
```
Comment (ctlai95) [inline .changes/next-release/feature-2152796a-9dd7-4889-9d00-3f56da0b2002.json:4]: Please remove this, we will add a single description when the feature branch is merged to `main` branch.

File: plugins/toolkit/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties [hunk 2]
```diff
@@ -743,9 +743,11 @@ codewhisperer.explorer.enabled=\   Enabled
 codewhisperer.explorer.inline_suggestions=Inline Suggestions
 codewhisperer.explorer.learn=Learn
 codewhisperer.explorer.pause_auto=Pause Auto-Suggestions
+codewhisperer.explorer.pause_auto_scans = Pause Auto Code Scans
 codewhisperer.explorer.paused=\   Paused
 codewhisperer.explorer.reconnect=Reconnect
 codewhisperer.explorer.resume_auto=Resume Auto-Suggestions
+codewhisperer.explorer.resume_auto_scans = Resume Auto Code Scans
```
Comment (ctlai95) [inline plugins/toolkit/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties:750]: Let's make these the same as vscode extension

```suggestion
codewhisperer.explorer.pause_auto_scans = Pause Auto-Scans
codewhisperer.explorer.paused=\   Paused
codewhisperer.explorer.reconnect=Reconnect
codewhisperer.explorer.resume_auto=Resume Auto-Suggestions
codewhisperer.explorer.resume_auto_scans = Resume Auto-Scans
```

**Review by ctlai95 — APPROVED** [review]

**Review by andrewyuq — APPROVED** [review]

**Review by justinmk3 — APPROVED** [review]

File: plugins/toolkit/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties [hunk 3]
```diff
@@ -779,9 +779,11 @@ codewhisperer.explorer.enabled=\   Enabled
 codewhisperer.explorer.inline_suggestions=Inline Suggestions
 codewhisperer.explorer.learn=Learn
 codewhisperer.explorer.pause_auto=Pause Auto-Suggestions
+codewhisperer.explorer.pause_auto_scans = Pause Auto-Scans
```
Comment (rli) [inline plugins/toolkit/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties:782]: scan or scans

**Review by rli — APPROVED** [review]

File: plugins/toolkit/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties (hunk 3 above)
Comment (ashishrp-aws) [reply plugins/toolkit/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties:782]: Scans. Like Tai used for VSCode: https://github.com/aws/aws-toolkit-jetbrains/pull/4211#discussion_r1543505646

**Pull request merged**